zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1163,6 +1163,12 @@ property, when available, is noted below.
     effect due to TLS handshake timeout when there are too many in-flight TLS
     handshakes. Set it to something like 250 is good enough to avoid herd effect.
 
+* *netty.server.earlyDropSecureConnectionHandshakes*
```

Comment:
I like this to be false by default. However, AFACT this behaviour was enabled in 3.7.0 by default. Maybe we should mention it in the documentation. (and also in the release doc of 3.7.1 and 3.8.0 if we don't forget)

Reply:
that "feature" was enabled in 3.6. this is way Pulsar and Pravega users are not able to upgrade ZK.

I can update the docs and explain the story.
I would like this patch to land to 3.6. 3.7 and 3.8

Reply:
updated